README.md

```diff
@@ -19,21 +19,39 @@ However, many things aren't implemented yet: for example, loops and switches are
 ![Sky shader](docs/assets/sky.jpg)
 
 ```rust
-#[spirv(entry = "fragment")]
-pub fn main_fs(input: Input<Vec4>, mut output: Output<Vec4>) {
-    let dir: Vec3 = input.load().truncate();
-
-    let cs_pos = Vec4(dir.0, -dir.1, 1.0, 1.0);
-    let ws_pos = {
-        let p = clip_to_world.mul_vec4(cs_pos);
-        p.truncate() / p.3
-    };
-    let dir = (ws_pos - eye_pos).normalize();
-    
+use spirv_std::{glam::{Vec3, Vec4, vec2, vec3}, storage_class::{Input, Output}};
+
+#[derive(Copy, Clone)]
+#[repr(C)]
+#[spirv(block)]
+pub struct ShaderConstants {
+    pub width: u32,
+    pub height: u32,
+}
```

Comment:
Not sure if we should include this in the readme, we removed it last time to keep the sample small.